college_preprocessor.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import pandas as pd
+import io, csv, os
+import redis
+import pika
+
+def SAT1(score):
+	if score != 'None':
+		if '-' not in score:
+			begin = score
+		else:
+			begin = score.split('-')[0]
+		if int(begin) > 100:
+			return begin
+	
+	return 'None'
+
+def SAT2(score):
+	if score != 'None':
+		if '-' not in score:
+			end = score
+		else:
+			end = score.split('-')[1]
+		if int(end) > 100:
+			return end
+	
+	return 'None'
+
+def ACT1(score):
+	if score != 'None':
+		if '-' not in score:
+			begin = score
+		else:
+			begin = score.split('-')[0]
+		if int(begin) < 100:
+			return begin
+
+	return 'None'
+
+def ACT2(score):
+	if score != 'None':
+		if '-' not in score:
+			end = score
+		else:
+			end = score.split('-')[1]
+		if int(end) < 100:
+			return end
+
+	return 'None'
+
+def ratio(rat):
+	if rat != 'None':
+		student = rat.split(':')[0]
+		faculty = rat.split(':')[1]
+		return float(student)/float(faculty)
+	return 'None'
+
+# csv file is located on the desktop
+csv_file = 'assets/colleges.csv'
+col_file = 'assets/Column Labels.txt'
```

Comment:
what does it mean for the csv file to be located on the desktop?

by convention our repo doesn't have spaces in file names